source/docs/casper/resources/tutorials/beginner/cep78/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Cep78
+slug: /cep78
+---
```

Comment:
Let us establish a standard where all tutorial pages have a Title with "Tutorial - xxx" as this catches the eye and differentiates the tutorials from the more prose-oriented pages.